terraform/environments/electronic-monitoring-data/server_access_capita.tf

```diff
@@ -1,3 +1,58 @@
+#------------------------------------------------------------------------------
```

Comment:
For the existing capita files i've moved the transfer user code from `transfer_server_capita.tf` to `server_access_capita.tf` - the code hasn't chnaged though.